src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1204,6 +1211,115 @@
+examples:
+  - name: Delete a mirror connected registry 'myconnectedregistry' from parent registry 'mycloudregistry' and skip verification.
+    text: |
+        az acr connected-registry delete --registry mycloudregistry 
+            --name myconnectedregistry --yes
+"""
+
+helps['acr connected-registry install-info'] = """
+type: command
+short-summary: Retrives information required for to activate a connected registry.
+examples:
+  - name: Prints the values requiered to activate a connected registry in json format
+    text: |
+        az acr connected-registry install --registry mycloudregistry 
+            --name myconnectedregistry 
+  - name: Prints the values in json format requiered to activate a connected registry and generates the sync token credentials.
+    text: |
+        az acr connected-registry install --registry mycloudregistry 
+            --name myconnectedregistry --fresh-install
+"""
+
+helps['acr connected-registry deactivate'] = """
+type: command
+short-summary: Deactivate a connected registry from Azure Container Registry.
+examples:
+  - name: Deactivate a connected registry 'myconnectedregistry' and skip verification.
+    text: |
+        az acr connected-registry deactivate --registry mycloudregistry 
+            --name myconnectedregistry --yes
+"""
+
+helps['acr connected-registry list'] = """
+type: command
+short-summary: Lists all the connected registries under the current parent registry.
+examples:
+  - name: Lists all the connected registries of 'mycloudregistry' in table format.
+    text: >
+        az acr connected-registry list --registry mycloudregistry --output table
+  - name: Lists only the inmediate children of 'mycloudregistry' in expanded form in a table.
+    text: >
+        az acr connected-registry list --registry mycloudregistry --no-children --output table
+  - name: Lists all the offspring of 'myconnectedregistry' in expanded form inside a table.
+    text: >
+        az acr connected-registry list --registry mycloudregistry --parent myconnectedregistry --output table
+"""
+
+helps['acr connected-registry list-client-tokens'] = """
+type: command
+short-summary: Lists all the client tokens associated to a specific connected registries.
+examples:
+  - name: Lists all client tokens of 'mymirroracr'.
+    text: >
+        az acr connected-registry list-client-tokens --registry mycloudregistry --name mymirroracr
+"""
+
+helps['acr connected-registry show'] = """
+type: command
+short-summary: Show connected registry details.
+examples:
+  - name: Show all the details of the 'mymirroracr' registry in table form.
+    text: |
+        az acr connected-registry show --registry mycloudregistry --name mymirroracr --output table
+"""
+
+helps['acr connected-registry update'] = """
+type: command
+short-summary: Update a connected registry for an Azure Container Registry.
+examples:
+  - name: Update the connected registry client Tokens.
```

Comment:
maybe little more specific update the connected registry to add/remove client tokens?